### ISSUE TITLE
Doubling <sqf:fixes/>

### ISSUE THREAD

**Issue by o-sapov** [oxygenxml/integrated-styleguide#3]
Why there is a <xsf:fixes/> section in the library.sch and quckFix-library.xml with same fixes?

**Comment by georgebina** [comment]
The library should inlclude to the quickFix-library.xml file just to allow the validate action on the library to find the fixes. Right now the part of the Schematron validation that checks SQF specific constraints does not follow the sch:include instructions, so the fixes were added directly there - we corrected that check to understand sch:include and sch:extends instructions so the quickFix-library.xml file can be included from library.sch rather than duplicating the fixes there - we will update the schema once we release a new oXygen version, that includes this fix. 

Best Regards,
George

**Comment by o-sapov** [comment]
Please, note also that in quickFix-library.xml no syntax highlighting works. Currently I'm going to code QuickFixes directly in the library.sch and then copy them to quickFix-library.xml by extractRules.xsl.

**Comment by georgebina** [comment]
Yes, we have an issue recorded to provide support for an SQF editor that will know that some attributes contain XPath expressions.
